### PR TITLE
5.0.10 update

### PR DESCRIPTION
**Pull request by razvanaldea89**

### CODE REVIEW

File: src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ public static function get_all_paths() {
 	 * @since 5.0.0
 	 */
 	public static function get_allowed_paths() {
-		$user_paths = DLM_Downloads_Path_Helper::get_all_paths();
+		$user_paths = self::get_all_paths();
```
Comment (cristeacosmin) [inline src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php:100]: https://php.watch/versions/8.2/partially-supported-callable-deprecation

File: src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php [hunk 2]
```diff
@@ -118,30 +118,26 @@ public static function get_allowed_paths() {
...
 	 * Enables a download path.
 	 *
-	 * @param  string  $path  The path string.
+	 * @param  string $path  The path string.
 	 *
 	 * @since 5.0.0
 	 */
 	public static function enable_download_path( $path ) {
-		$paths = DLM_Downloads_Path_Helper::get_all_paths();
+		$paths = self::get_all_paths();
```
Comment (cristeacosmin) [inline src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php:140]: https://php.watch/versions/8.2/partially-supported-callable-deprecation

**Review by cristeacosmin — COMMENTED** [review]
nu stiu daca pr-u asta e mare ca si features sau ca s-a adaugat un punct pe fiecare comment

File: src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php (hunk 1 above)
Comment (razvanaldea89) [reply src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php:100]: O ia razna phpcs-ul daca schimb 
image

Comment (razvanaldea89) [reply src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php:100]: si cred ca phpcs-ul a si schimbat aici

Comment (razvanaldea89) [reply src/Admin/DownloadPaths/class-dlm-downloads-path-helper.php:100]: legat de punct in comment, phpcs again. dar sunt si fixuri/schimbari destule.

**Review by cristeacosmin — APPROVED** [review]

**Pull request merged**